appinventor/appengine/src/com/google/appinventor/client/style/neo/ImagesNeo.java

```diff
@@ -0,0 +1,938 @@
+package com.google.appinventor.client.style.neo;
+
+import com.google.appinventor.client.Images;
+import com.google.appinventor.client.Ode;
+import com.google.gwt.resources.client.ImageResource;
+
+public interface ImagesNeo extends Images {
```

Comment:
Maybe we should only override the methods here of images that are actually changed?